src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToReturnViewModelMap.cs

```diff
@@ -64,7 +64,8 @@ public ReturnViewModel Map(ReturnData source)
                 tonnageUtilities.CheckIfTonnageIsNull(NonObligatedTonnageTotal),
                 tonnageUtilities.CheckIfTonnageIsNull(NonObligatedTonnageTotalDcf),
                 AatfObligatedData,
-                source.ReturnOperatorData);
+                source.ReturnOperatorData,
```

Comment:
Looks like we missed some unit tests for this class... could you possible add some :) ta...